src/navigations/KeyboardWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {ViewStyle, KeyboardAvoidingView, Platform, ScrollView} from 'react-native';
+import dimen from '../utils/dimen';
+
+type Props = {
+  children: any;
+  keyboardVerticalOffset?: number;
+  contentContainerStyle?: ViewStyle;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Props` type definition is not strict enough. The `children` prop is typed as `any`, which is too permissive and could lead to runtime errors if misused. It's better to use `ReactNode` or `ReactElement` for children. Also, the `keyboardVerticalOffset` prop should be strictly a number, not optional, to ensure consistent behavior across different usages of this component.
```diff
- type Props = {
-   children: any;
-   keyboardVerticalOffset?: number;
-   contentContainerStyle?: ViewStyle;
- };
+ type Props = {
+   children: React.ReactNode;
+   keyboardVerticalOffset: number;
+   contentContainerStyle?: ViewStyle;
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->